client/src/containers/pages/Home/tests/__snapshots__/Home.test.tsx.snap

```diff
@@ -214,6 +214,80 @@ exports[`containers/pages/Home renders Home correctly & changes Title of page: H
             </Link>
           </div>
         </Col>
+        <Col
```

Comment:
Is this test just showing the loading icon?

Reply:
i just added a button on home page so that we can navigate to this (LOCATIONS UNIT) component.

this test is showing those buttons as well as the animation node they have